alerting/build.gradle

```diff
@@ -115,6 +115,7 @@ dependencies {
     api project(":alerting-core")
     implementation "com.github.seancfoley:ipaddress:5.3.3"
 
+    testImplementation "org.antlr:antlr4-runtime:${versions.antlr4}"
```

Comment:
why is this required?

Reply:
This causes the integration tests to fail since its missing this dependency. This is because it is used by the Painless scripts and its caused by the BucketSelectorExtAggregator. I tried having this change in common-utils, but it wouldn't help.